### PR TITLE
data:Hrishi5111998

### PR DESCRIPTION
**Pull request by Hrishi5111998**
## Changes proposed 
     links added to profile 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3141"></a>

### CODE REVIEW

File: data/Hrishi5111998 [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Hrishikesh Tule",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web developer **Digital Nomad**. open source enthusiast",
+  "avatar": "https://github.com/Hrishi5111998.png",
+}
```
Comment (shivaabhishek07) [inline data/Hrishi5111998:7]: This flower braces should be removed!

File: data/Hrishi5111998 [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+{
```
Comment (shivaabhishek07) [inline data/Hrishi5111998:8]: here before flower brace, you should add link attribute!
And after for every list object you should have both opening and closing flower braces.
Expected to be like this
`"link"=[
     {
      "name": "Twitter",
      "url": "https://twitter.com/@hrishikesh7030",
      "icon": "FaTwitter"
    },
   {
    "name": "Instagram",
      "url": "https://instagram.com/@hrishikesh2412",
      "icon": "FaInstagram"
      },
     {
      "name": "LinkedIn",
      "url": "https://linkedin.com/in/hrishikesh-tule",
      "icon": "FaLinkedIn"
      },
     {
      "name": "Github",
      "url": "https://github.com/Hrish5111998",
      "icon": "FaGithub"
      }
  ]
`

**Review by shivaabhishek07 — CHANGES_REQUESTED** [review]
Overall syntax is not well versed!

**Comment by Hrishi5111998** [comment]
> Hrishi5111998

can you please elaborate

**Comment by Hrishi5111998** [comment]
> Overall syntax is not well versed!

suggested changes done please check!

**Review by accodes21 — COMMENTED** [review]
Syntax still has issues. The indentation is incorrect

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Missing file extension, please add `.json`

**Comment by Hrishi5111998** [comment]
> Missing file extension, please add `.json`

'.json' extension added. please check and merge request.

File: data/Hrishi5111998.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/hrishikesh-tule",
+      "icon": "FaLinkedIn"
+      },
+      {
+      "name": "Github",
+      "url": "https://github.com/Hrish5111998",
+      "icon": "FaGithub"
+      },
```
Comment (eddiejaoude) [inline data/Hrishi5111998.json:27]: This trailing comma is not required

Comment (Hrishi5111998) [reply data/Hrishi5111998.json:27]: ok thanks for suggestion I will get it done.

**Comment by Hrishi5111998** [comment]
> I think 1 small tweak remaining, then we can get it merged

can you tell what's that tweak. please

**Comment by swagataroy30** [comment]
Why did you remove the commas? You need to add commas at the end of line 2,3,4,5,6,12,17,22.

File: data/Hrishi5111998.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    {
+      "name": "Instagram"
+      "url": "https://instagram.com/@hrishikesh2412"
+      "icon": "FaInstagram"
+      }
+      {
+      "name": "LinkedIn"
+      "url": "https://linkedin.com/in/hrishikesh-tule"
+      "icon": "FaLinkedIn"
```
Comment (swagataroy30) [inline data/Hrishi5111998.json:21]: Fix the typo, it's "FaLinkedin"

**Comment by Pradumnasaraf** [comment]
The profile is already added in https://github.com/EddieHubCommunity/LinkFree/pull/3416. closing this duplicate PR.

**Pull request closed without merging**